app-profile-jee-vanilla/README.md

```diff
@@ -1,37 +1,25 @@
-app-profile-jee-vanilla: JSP Profile Application
-================================================
+## app-profile-jee-vanilla: JSP Profile Application
 
 Level: Beginner  
 Technologies: JavaEE  
 Summary: JSP Profile Application with Basic Authentication  
 Target Product: <span>Keycloak</span>, <span>WildFly</span>  
-Source: <https://github.com/keycloak/keycloak-quickstarts>  
+Source: <https://github.com/keycloak/keycloak-quickstarts>
 
+### What is it?
 
-What is it?
------------
+The `app-profile-jee-vanilla` quickstart demonstrates how to change a JavaEE application that is secured with basic authentication without any changes to the WAR itself. Changing the authentication method and injecting the configuration is done automatically by the <span>Keycloak</span> client adapter subsystem.
```

Comment:
The change of authentication method is not done automatically with Elytron OIDC adapter. This should be rephrased.